lib/reporting/report_row_builder.rb

```diff
@@ -124,5 +127,11 @@ def format_date(value)
     def format_numeric(value)
       number_with_delimiter(value)
     end
+
+    def format_percentage(value)
+      return '' if value.blank?
+
+      I18n.t('admin.reports.percentage', value: value * 100)
+    end
```

Comment:
I just discovered the `number_to_percentage` helper included in Rails: https://guides.rubyonrails.org/i18n.html#action-view-helper-methods

Would that be useful here?

Reply:
I tried to use it, but it looks like it's part of [Action View Helper Methods](https://guides.rubyonrails.org/i18n.html#action-view-helper-methods) and it's not available in the current context.

Reply:
Okay. Not as convenient. It's still possible though:

```rb
ApplicationController.helpers.number_to_percentage(value * 100, precision: 2)
```